client/src/app/+admin/system/logs/logs.component.html

```diff
@@ -30,14 +31,16 @@
 
   <my-select-tags *ngIf="!isAuditLog()" i18n-placeholder placeholder="Filter logs by tags" [(ngModel)]="tagsOneOf" (ngModelChange)="refresh()"></my-select-tags>
 
-  <my-button i18n-label label="Refresh" icon="refresh" (click)="refresh()"></my-button>
+  <my-button *ngIf="!isAuditLog() || isAuditLogsEnabled" i18n-label label="Refresh" icon="refresh" (click)="refresh()"></my-button>
 </div>
 
 <div class="logs">
   <div *ngIf="loading" i18n>Loading...</div>
 
   <div #logsElement>
-    <div *ngIf="!loading && logs.length === 0" i18n>No log.</div>
+    <div *ngIf="isAuditLog && !isAuditLogsEnabled" i18n>Audit logs disabled.</div>
```

Comment:
`isAuditLog` is a function so `isAuditLog()`

Also transform `isAuditLogsEnabled` to a function. `is..` is for functions, if you still want to have a variable then prefer `auditLogsEnabled`. But it's better to have a function to not mix variables/functions if we can avoid